code/modules/mob/living/silicon/decoy/decoy.dm

```diff
@@ -31,16 +34,19 @@
 
 	update_stat()
 
-/mob/living/silicon/decoy/death(gibbed, deathmessage = "sparks up and falls silent...")
+/mob/living/silicon/decoy/death()
 	set waitfor = 0
-	if(stat == DEAD) return FALSE
+	if(stat == DEAD) 
+		return FALSE
 	icon_state = "hydra-off"
+	src.visible_message("<b>\The [src.name]</b> sparks up and falls silent...")
 	sleep(20)
 	explosion(loc, -1, 0, 8, 12)
-	return ..()
+	stat = DEAD
```

Comment:
Why are you removing the parent call, you should be preferably solving this in such a way that you keep it